### DISCUSSION TITLE
Cannot read property 'register' of null

### DISCUSSION THREAD

**Discussion by EliasPiscitelli** [react-hook-form/react-hook-form#5133] [Q&A]
I am following the example as the page tells me and it throws me that error I am working with next js, I already looked in several places and I cannot find the solution
![image](https://user-images.githubusercontent.com/65465756/117743319-f538e480-b1dc-11eb-8aaa-4011c9559bfe.png)
![image](https://user-images.githubusercontent.com/65465756/117743334-fcf88900-b1dc-11eb-9bc8-d9ab195f49d3.png)
![image](https://user-images.githubusercontent.com/65465756/117743356-0c77d200-b1dd-11eb-9203-b3a4bb34c68a.png)
![image](https://user-images.githubusercontent.com/65465756/117743388-21ecfc00-b1dd-11eb-8e59-602e937417f3.png)

**Comment by bluebill1049** [comment]
Is ur app wrapped with FormProvider ?

**Reply by EliasPiscitelli** [reply]
Not only the component that I use and that I want to reach, in the examples it does not wrap app

**Comment by EliasPiscitelli** [comment]
this is the other component
![image](https://user-images.githubusercontent.com/65465756/117745046-1c44e580-b1e0-11eb-8e19-2122904c276b.png)
this is the parent component
![image](https://user-images.githubusercontent.com/65465756/117745219-73e35100-b1e0-11eb-9393-6d8814272819.png)

change the formProvider to app and fixed the null error
![image](https://user-images.githubusercontent.com/65465756/117745103-3979b400-b1e0-11eb-81f3-e12f3d3edc9b.png)
But when loading the data and giving it submit, it does not bring me the information of the other component
